health_check/templates/health_check/index.html

```diff
@@ -2,31 +2,73 @@
 <head>
   <title>{% block title %}System status{% endblock title %}</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <style>
+    html, body {
+      margin: 0 auto;
+      font-family: Verdana, sans-serif;
```

Comment:
We use two different fonts, is that on purpose? Maybe we just default to the system font. Perhaps even a monospaced font?

Reply:
It wasn't, no. I think I just borrowed it from the old styles. I've updated it to use just 1 font, the system `sans-serif` font.